airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -2630,6 +2630,7 @@ class BigQueryInsertJobOperator(GoogleCloudBaseOperator):
     :param result_retry: How to retry the `result` call that retrieves rows
     :param result_timeout: The number of seconds to wait for `result` method before using `result_retry`
     :param deferrable: Run operator in the deferrable mode
+    :param poll_interval: polling period in seconds to check for the status of job
```

Comment:
```suggestion
    :param poll_interval: polling period in seconds to check for the status of job. Defaults to 4 seconds.
```
WDYT about adding the default value to the docstring?

Reply:
make sense